pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1748,8 +1748,8 @@ func TestReconcileInvalidTaskRuns(t *testing.T) {
 
 			// Check actions and events
 			actions := clients.Kube.Actions()
-			if len(actions) != 3 || actions[0].Matches("namespaces", "list") {
```

Comment:
Note that all of these checks were previously incorrect because they were missing `!` and had their arguments transposed. 🙃